virtool/uploads/utils.py

```diff
@@ -65,9 +75,7 @@ async def naive_writer(
         pass
 
     async with aiofiles.open(path, "wb") as f:
-        while True:
-            chunk = await file.read_chunk(CHUNK_SIZE)
-
+        async for chunk in chunker:
             if not chunk:
                 break
```

Comment:
This should not be necessary. Once chunks are exhausted, the end of the iterator will be reached an you'll exit this for loop.

I don't believe the iterator ever ends as implemented with `while True` currently.

```suggestion
            if not chunk:
                break
```